### ISSUE TITLE
Snapshot Metals can't run scala-cli project

### ISSUE THREAD

**Issue by ckipp01** [scalameta/metals#4646]
### Describe the bug

Current when you're using the latest snapshot of Metals with scala-cli you can't run a file even though you get a code lens due to the change in https://github.com/scalameta/metals/pull/4628.

To reproduce this create a file with the following:

```scala
@main def foo() =
  println("hello")
```

Trigger the code lens to run the project and see the following in the logs:

```
Caused by: java.lang.IllegalArgumentException: scala-cli 0.1.17 does not support scala-debug-adpater 2.x
        at scala.meta.internal.metals.debug.DebugProvider.$anonfun$start$12(DebugProvider.scala:162)
        at scala.meta.internal.metals.debug.DebugServer.loop$1(DebugServer.scala:21)
        at scala.meta.internal.metals.debug.DebugServer.listen$lzycompute(DebugServer.scala:34)
        at scala.meta.internal.metals.debug.DebugServer.listen(DebugServer.scala:19)
        at scala.meta.internal.metals.debug.DebugProvider.start(DebugProvider.scala:180)
        at scala.meta.internal.metals.debug.DebugProvider.$anonfun$start$5(DebugProvider.scala:108)
        at scala.concurrent.impl.Promise$Transformation.run(Promise.scala:470)
        ... 3 more
```
The user doesn't get any indications that anything is wrong either.

### Expected behavior

This should probably be surfaced to the user. Also is this happening because the Bloop fork for scala-cli hasn't been updated?

### Operating system

macOS

### Editor/Extension

Nvim (nvim-metals)

### Version of Metals

0.11.9+146-25674662-SNAPSHOT

### Extra context or search terms

```
## Metals Info
  - Metals Server version: 0.11.9+146-25674662-SNAPSHOT
  - Build server currently being used is scala-cli v0.1.17.
  - Metals Java: 17.0.5 from Eclipse Adoptium located at /Users/ckipp/.sdkman/candidates/java/17.0.5-tem
```

**Comment by tgodzik** [comment]
Thanks for reporting. Actually, scala-cli bloop fork is regularly updated and scala debug Adapter 1.x was released way over a year ago. I can take a look next week

**Comment by ckipp01** [comment]
Fixed via #4649 4649
